### ISSUE TITLE
User reviews list shows reviews from all the games

### ISSUE THREAD

**Issue by Arcadius2006** [LittleBigRefresh/Refresh#642]
pretty self explanatory: LBP2 shows LBP3, LBPVita, and beta build reviews

i have not tested this on LBP3 yet, and i straight up can't test it on vita, but i'm pretty sure the issue affects those games too

**Comment by Toastbrot236** [comment]
Yes, the original server also showed all reviews from all games.

**Comment by Arcadius2006** [comment]
no, not the on-level reviews, the profile ones - they also show the LBP3 and vita levels despite you not being able to play those levels in LBP2

**Comment by Arcadius2006** [comment]
i mean we're also not taking into account that vita was also on an entirely seperate server, unlike what bonsai does
